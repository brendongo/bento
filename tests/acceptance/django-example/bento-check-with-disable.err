Running Bento checks...


⚠ 1 finding(s) in 0.62 s
◦ To suppress all findings, run $ bento archive.
