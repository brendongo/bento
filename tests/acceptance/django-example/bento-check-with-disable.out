flake8 not-executable https://github.com/xuhdev/flake8-executable/blob/master/README.md#flake8-executable
     > manage.py:1
     ╷
    1│   #!/usr/bin/env python
     ╵
     = Shebang is present but the file is not executable.

flake8:
  not-executable     1▕▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬

